src/modes/play/levels/2.js

```diff
@@ -119,6 +120,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
   mobs: {
     p: '',
     g: '',
+    f: (g, m, x, y) => new GrowthFungus(g, m, x, y, 'flee'),
```

Comment:
You can pass the "flee" flag to it, which affects how the brain works... i.e. Whether or not it dispatches actions to make it move around.